components/common.js

```diff
@@ -6,7 +6,7 @@
         if (window.NodeList.prototype.forEach) {
             return nodes.forEach(callback);
         }
-        for (var i = 0; i < nodes.length; i++) {
+        for (let i = 0; i < nodes.length; i++) {
```

Comment:
We can keep `var` in all of the client-side JS for maximum compatibility. Can update ESLint config to allow this on client-side JS files only. This is all JS under the `components/` directory here.

Reply:
We still want to roll back these changes that were due to Node rules being applied to client-side JS